src/Nether.Analytics.DataLake/Scheduler/DateTimeUtilities.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace Nether.Analytics.DataLake
+{
+    public static class DateTimeUtilities
+    {
+        /// <summary>
+        /// Parses a new DateTime from format yyyyMMdd-HHmm
+        /// </summary>
+        /// <param name="from">DateTime in string format yyyyMMdd-HHmm</param>
+        /// <returns>A new DateTime</returns>
+        public static DateTime FromSpecialString(string from)
```

Comment:
We should try to use more specific words than "specal string" since that can mean anything.